client/blocks/inline-help/popover.jsx

```diff
@@ -168,18 +199,51 @@ class InlineHelpPopover extends Component {
 }
 
 function mapStateToProps( state ) {
+	const siteId = getSelectedSiteId( state );
+	const currentRoute = getCurrentRoute( state );
+	const classicRoute = currentRoute.replace( '/gutenberg/', '' );
+
 	return {
 		searchQuery: getSearchQuery( state ),
 		selectedSite: getHelpSelectedSite( state ),
 		selectedResult: getInlineHelpCurrentlySelectedResult( state ),
+		selectedEditor: getSelectedEditor( state, siteId ),
+		classicUrl: `/${ classicRoute }`,
+		siteId,
 	};
 }
 
-export default connect(
-	mapStateToProps,
-	{
+function mapDispatchToProps( dispatch ) {
+	return {
+		optOut: siteId => {
+			dispatch(
+				withAnalytics(
+					composeAnalytics(
+						recordGoogleEvent(
+							'Gutenberg Opt-Out',
+							'Clicked "Switch to the classic editor" in the help popover.',
+							'Opt-In',
+							false
+						),
+						recordTracksEvent( 'calypso_gutenberg_opt_in', {
+							opt_in: false,
+						} ),
+						bumpStat( 'gutenberg-opt-in', 'Calypso Help Opt Out' )
+					),
+					setSelectedEditor( siteId, 'classic' )
+				)
+			);
+		},
 		recordTracksEvent,
 		selectResult,
 		resetContactForm: resetInlineHelpContactForm,
```

Comment:
Note that we should either stick with keeping an object or function return. we'll run into trouble here since dispatch isn't used for these. Object notation is shorthand for assuming each prop is an action creator.

> (Object or Function): If an object is passed, each function inside it is assumed to be a Redux action creator. An object with the same function names, but with every action creator wrapped into a dispatch call so they may be invoked directly, will be merged into the component’s props.

https://github.com/reduxjs/react-redux/blob/master/docs/api.md

Reply:
Ah, makes sense. TY!